csharp/ql/src/experimental/Security Features/Serialization/DataSetSerialization.qll

```diff
@@ -29,17 +26,13 @@ class DataSetOrTable extends DataSetOrTableRelatedClass {
  */
 class ClassWithDataSetOrTableMember extends DataSetOrTableRelatedClass {
   ClassWithDataSetOrTableMember() {
-    exists( Property p |
-      p = this.getAProperty() |
-      p.getType() instanceof DataSetOrTable
-    ) or exists ( AssignableMember am |
-      am = this.getAField() or
-      am = this.getAMember() |
-      am.getType() instanceof DataSetOrTable
-    ) or exists( Property p |
-      p = this.getAProperty() |
+    exists(Property p | p = this.getAProperty() | p.getType() instanceof DataSetOrTable)
```

Comment:
This line can be removed as it is included by the disjunct below (`getAMember()` includes properties).

Reply:
Fixing it